compression/compression.go

```diff
@@ -0,0 +1,96 @@
+// Package compression is a horrible hack & should be replaced
+// as soon as humanly possible
```

Comment:
what's horrible about it and what should it be replaced with?

Reply:
lololol old, ded comment. will remove